docs/dev/table/connectors/kafka.md

```diff
@@ -0,0 +1,175 @@
+| 0.10.x              | `flink-connector-kafka-010{{site.scala_version_suffix}}`  | [Download](https://repo.maven.apache.org/maven2/org/apache/flink/flink-connector-kafka-010{{site.scala_version_suffix}}/{{site.version}}/flink-connector-hbase{{site.scala_version_suffix}}-{{site.version}}.jar) |
+
+<span class="label label-danger">Attention</span> The ``0.10`` sink does not support exactly-once writes to Kafka.
+{% else %}
+
+The dependency table is only available for stable releases.
+
+{% endif %}
+
+Flink's streaming connectors are not currently part of the binary distribution.
+See how to link with them for cluster execution [here]({{ site.baseurl}}/dev/projectsetup/dependencies.html).
+
+How to create a Kafka table
+----------------
+<div class="codetabs" markdown="1">
+<div data-lang="SQL" markdown="1">
+{% highlight sql %}
+CREATE TABLE kafkaTable (
+ ...
+) WITH (
+ 'connector' = 'kafka',
+ 'topic' = 'topic_name',
+ 'properties.bootstrap.server' = 'localhost:9092',
+ 'properties.group.id' = 'testGroup',
+ 'format' = 'csv',
+ 'scan.startup.mode' = 'group-offsets'
+)
+{% endhighlight %}
+</div>
+</div>
+
+Connector Options
+----------------
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 25%">Option</th>
+        <th class="text-center" style="width: 8%">Required</th>
+        <th class="text-center" style="width: 7%">Default</th>
+        <th class="text-center" style="width: 10%">Type</th>
+        <th class="text-center" style="width: 50%">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>connector</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Specify what connector to use, for Kafka the options are: 'kafka', 'kafka-0.11', 'kafka-0.10'.</td>
+    </tr>
+    <tr>
+      <td><h5>topic</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required topic name from which the table is read.</td>
+    </tr>
+    <tr>
+      <td><h5>properties.bootstrap.servers</h5></td>
+      <td>required</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Required Kafka server connection string.</td>
+    </tr>
+    <tr>
+    <td><h5>format</h5></td>
+    <td>required</td>
+    <td style="word-wrap: break-word;">(none)</td>
+    <td>String</td>
+    <td>Kafka connector requires to specify a format,
+    the supported formats are 'csv', 'json' and 'avro'.
+    Please refer to [Table Formats]({{ site.baseurl }}/dev/table/connect.html#table-formats) section for more details.
+    </td>
+        </tr>
+    <tr>
+      <td><h5>properties.group.id</h5></td>
+      <td>optional</td>
```

Comment:
I prefer to mark it as `required by source` and move it after `properties.bootstrap.servers`.